### PR TITLE
TASK: fix typos in docs using codespell

### PR DESCRIPTION
**Pull request by crydotsnake**
_Please check the following:_

- [x] Do the tests still pass?[^1]
- [x] Does the code comply with the style guide? 
    - [x] Run `make lint` from the Wagtail root. 
- [ ] For Python changes: Have you added tests to cover the new/fixed behaviour?
- [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    - [ ] **Please list the exact browser and operating system versions you tested**:
    - [ ] **Please list which assistive technologies [^3] you tested**: 
- [ ] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

I fixed some typos in the documentation using codespell

- https://github.com/codespell-project/codespell

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

File: docs/advanced_topics/deploying.rst [hunk 1]
```diff
@@ -60,7 +60,7 @@ Wagtail follows :doc:`Django's conventions for managing uploaded files <django:t
...
...'s ``MEDIA_ROOT``. If all your documents are public, you can set the ``WAGTAILDOCS_SERVE_METHOD`` to ``direct`` and configure your web server to serve the files itself. However, if you use  Wagtail's :ref:`Collection Privacy settings <collection_privacy_settings>` to restrict access to some or all of your documents, you may or may not want to configure your web server to serve the documents directly. The default setting is ``redirect`` which allows Wagtail to perform any configured privacy checks before offloading serving the actual document to your web server or CDN. This means that wagtail constructs document links that pass through Wagtail, but the final url in the user's browser is served directly by your web server. If a user bookmarks this url, they will be able to access the file without passing through Wagtail's privacy checks. If this is not acceptable, you may want to set the ``WAGTAILDOCS_SERVE_METHOD`` to ``serve_view`` and configure your web server so it will not serve document files itself. If you are serving documents from the cloud and need to enforce privacy settings, you should make sure the documents are not publicly accessible using the cloud service's file url.
```
Comment (lb-) [inline docs/advanced_topics/deploying.rst:63]: This was already fixed in another PR - see 5c894345b993dca8b68178b5965a2ccce05045a0

File: docs/advanced_topics/formbuilder_routablepage_redirect.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ By default `wagtail.contrib.forms.models.FormPage` success responses don't do th
 
 Instead of rendering the `render_landing_page` content in the POST response, we will redirect to a `route` of the `FormPage` instance at a child URL path.
 The content will still be managed within the same form page's admin.
-This approach uses the additonal contrib module `wagtail.contrib.routable_page`.
+This approach uses the additional contrib module `wagtail.contrib.routable_page`.
```
Comment (lb-) [inline docs/advanced_topics/formbuilder_routablepage_redirect.md:8]: Same here 5c894345b993dca8b68178b5965a2ccce05045a0

File: docs/contributing/security.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Mail sent to that address reaches a subset of the core team, who can forward sec
 
 Once you've submitted an issue via email, you should receive an acknowledgement from a member of the security team within 48 hours, and depending on the action to be taken, you may receive further followup emails.
 
-If you want to send an encrypted email (optional), the public key ID for <security@wagtail.org> is `0xbed227b4daf93ff9`, and this public key is available from most commonly-used keyservers.
+If you want to send an encrypted email (optional), the public key ID for <security@wagtail.org> is `0xbed227b4daf93ff9`, and this public key is available from most commonly-used key servers.
```
Comment (lb-) [inline docs/contributing/security.md:14]: I think key servers is more correct here.

**Comment by lb-** [comment]
@crydotsnake thanks for these fixes, as noted some were already in.

I have updated your remote branch and added your name to the contributors, thanks for taking the time to submit these fixes!

I have added your name as `Simon Krull`, please let me know if you wanted it listed differently.

**Comment by crydotsnake** [comment]
Hello @lb- 

Thanks for your replys!

Where you have listed my Name? 🤔

**Comment by lb-** [comment]
See the files changed on this PR - `CONTRIBUTORS.rst`

For small typo fixes in docs we don't add these to the changelog.

**Pull request merged**